javaparser-core-testing/src/test/java/com/github/javaparser/printer/PrettyPrinterTest.java

```diff
@@ -225,9 +234,14 @@ void prettyAlignMethodCallChainsIndentsArgumentsWithBlocksCorrectly() {
 
         CompilationUnit cu = parse("class Foo { void bar() { a.b.c.d.e; a.b.c().d().e(); a.b.c().d.e(); foo().bar().baz(boo().baa().bee()).bam(); foo().bar().baz(boo().baa().bee()).bam; foo().bar(Long.foo().b.bar(), bam).baz(); foo().bar().baz(foo, () -> { boo().baa().bee(); }).baz(() -> { boo().baa().bee(); }).bam(() -> { boo().baa().bee(); }); } }");
         Indentation indentation = new Indentation(TABS_WITH_SPACE_ALIGN, 1);
-        String printed = new PrettyPrinter(new PrettyPrinterConfiguration().setColumnAlignFirstMethodChain(true).setColumnAlignParameters(true).setIndentation(indentation))
-                .print(cu);
 
+        ConfigurablePrinter config = new PrinterConfiguration()
+                .addOption(ConfigOption.COLUMN_ALIGN_FIRST_METHOD_CHAIN)
```

Comment:
Hm, the explicit configuration methods are designed to have the same API approach as ParserConfiguration, and to aid in discoverability (I can look at the methods on the configuration class, instead of looking at the configuration class, then looking at the ConfigOption class - an extra step.)

Reply:
Yes we lose in semantics but we gain in extensibility.

Reply:
Yes, and you lose consistency and ease of use.